components/clientComponents/forms/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+import React from "react";
+import { InputFieldProps } from "@lib/types";
+import classnames from "classnames";
+import { useField } from "formik";
+import { ErrorMessage } from "@clientComponents/forms";
+import { useCombobox } from "downshift";
+import { cn } from "@lib/utils";
+
+interface ComboboxProps extends InputFieldProps {
+  choices?: string[];
+}
+
+export const Combobox = (props: ComboboxProps): React.ReactElement => {
+  const { id, name, className, choices = [], required, ariaDescribedBy } = props;
+  const classes = classnames("gc-combobox", className);
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [field, meta, helpers] = useField(props);
+  const { setValue } = helpers;
+
+  const [items, setItems] = React.useState(choices);
+  const { isOpen, getMenuProps, getInputProps, highlightedIndex, getItemProps, selectedItem } =
+    useCombobox({
+      onInputValueChange({ inputValue }) {
+        setItems(
+          choices.filter((choice) => {
+            return inputValue ? choice.toLowerCase().includes(inputValue.toLowerCase()) : true;
+          })
+        );
+      },
+      items,
+      onSelectedItemChange({ selectedItem }) {
+        setValue(selectedItem);
+      },
+    });
+
+  return (
+    <>
```

Comment:
We should be able to remove this fragment yes?

Reply:
yup